datafusion/expr/src/lib.rs

```diff
@@ -66,20 +66,7 @@ pub use function::{
     StateTypeFunction,
 };
 pub use literal::{lit, lit_timestamp_nano, Literal, TimestampLiteral};
-pub use logical_plan::{
-    builder::{
-        build_join_schema, union, wrap_projection_for_join_if_necessary, UNNAMED_TABLE,
-    },
-    Aggregate, Analyze, CreateCatalog, CreateCatalogSchema, CreateExternalTable,
-    CreateMemoryTable, CreateView, CrossJoin, DescribeTable, Distinct, DmlStatement,
-    DropTable, DropView, EmptyRelation, Explain, Extension, Filter, Join, JoinConstraint,
-    JoinType, Limit, LogicalPlan, LogicalPlanBuilder, Partitioning, PlanType, Prepare,
-    Projection, Repartition, SetVariable, Sort, Statement, StringifiedPlan, Subquery,
-    SubqueryAlias, TableScan, ToStringifiedPlan, TransactionAccessMode,
-    TransactionConclusion, TransactionEnd, TransactionIsolationLevel, TransactionStart,
-    Union, Unnest, UserDefinedLogicalNode, UserDefinedLogicalNodeCore, Values, Window,
-    WriteOp,
-};
+pub use logical_plan::*;
```

Comment:
this code was just re-exporting all the names in logical_plan anyways so I reduced the duplication